File: Java/JavaPlayground/Classes.java

public class Classes {
    public static void main(String[] args) {
        Circle newCircle = new Circle();

        newCircle.setColor("orange");
        newCircle.setRadius(15);
        System.out.println("The circumference of the Circle is " + newCircle.getCircumference());
    }
}

File: C++/CodePlayground/Classes.cpp
#include <iostream>
#include "Circle.h"

int main()
{
    Circle newCircle;

    newCircle.setColor("orange");
    newCircle.setRadius(15);
    std::cout << "The circumference of this circle is " << newCircle.getCircumference() << std::endl;
    return 0;
}